docs/UserGuide.md

```diff
@@ -38,15 +42,15 @@ We store this data locally on your device. This means that you can benefit from
 
 3. Copy the file to the folder you want to use as the _home folder_ for your checkUp instance.
 
-4. Navigate to the home folder with `cd PATH_TO_FOLDER` in your command line interface.
+4. Navigate to the home folder with `cd PATH_TO_FOLDER` through your command line interface.
 
 5. Run `java -jar checkUp.jar` to start the app. The GUI similar to the below image should appear after startup.
 
 6. Note how the app contains some sample data.<br>
    ![Ui](images/ug-images/Ui-explanations.png)
 
-7. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will
-   open the help window.<br>
+7. Type your desired command into the command box and press Enter to execute it. e.g. typing **`help`** and pressing
+   Enter will open the help window.<br>
```

Comment:
Could you change the word "Enter" to include angle brackets `<Enter>` so readers know that it is a button? Otherwise it would just be a capitalisation and that would be weird.

Reply:
Sounds good, this has been addressed!